opba-banking-rest-impl/src/test/java/de/adorsys/opba/tppbanking/impl/controller/TestBankSearchPerformance.java

```diff
@@ -81,13 +81,13 @@ private Runnable getBanksEndpoint(StatisticService statisticService, List<String
             long start = System.currentTimeMillis();
             try {
                 MvcResult mvcResult = mockMvc.perform(
-                        get("/v1/banking/search/bank-search")
+                        MockMvcRequestBuilders.get("/v1/banking/search/bank-search")
```

Comment:
Import MockMvcRequestBuilders statically?

Reply:
Idea changed it automatically. 
fixed